src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/Validation/OnboardedCountriesCBORDataFake.swift

```diff
@@ -0,0 +1,26 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import SwiftCBOR
+
+public var onboardedCountriesCBORDataFake: Data {
+    let cborCountries = CBOR.array(
+        [
+            CBOR.utf8String("DE"),
+            CBOR.utf8String("FR")
+        ]
+    )
+    return Data(cborCountries.encode())
+}
+
+public var onboardedCountriesCorruptCBORDataFake: Data {
+    let cborCountries = CBOR.array(
+        [
+            CBOR.null,
+            CBOR.unsignedInt(42)
+        ]
+    )
+    return Data(cborCountries.encode())
+}
```

Comment:
Do we really need global variables for a very limited scope? If this is just used in the test below, `internal` is fine.

Reply:
yeah, true 👍